tests/NuGet.Services.AzureSearch.Tests/SearchService/AzureSearchServiceFacts.cs

```diff
@@ -16,6 +16,23 @@ public class AzureSearchServiceFacts
     {
         public class V2SearchAsync : BaseFacts
         {
+            [Fact]
+            public async Task SearchIndexAndEmptyOperation()
```

Comment:
Do you think it's worth testing that no search/hijack operations were performed? I don't feel strongly about adding this, feel free to skip

Reply:
Sure, sounds good.